uri-schemes.md

```diff
@@ -161,16 +161,21 @@ https://i.delta.chat/#EEA98F87742EF2FD6C23677F1E1142828C202998&a=demo.fn8hk%40fi
 
 ### Syntax
 
-currently this only accepts a url that gives back account setup information as a json object.
-see the [mailadm](https://github.com/deltachat/mailadm) project for more details as this api was designed for use with burner account qr codes generated by mailadm.
+This scheme contains either the domain name of a chatmail server or an HTTPS URL.
 
-```
-DCACCOUNT:https://example.org/new_email?t=1w_7wDjgjelxeX884x96v3
-```
+If domain name is used, e.g. as `DCACCOUNT:example.org`,
+then the client should generate a 9-character alphanumeric lowercase username
+and at least 30-character-long password using at least all alphanumeric characters,
```

Comment:
"all alphanumeric characters" literally means 62 characters, i think this wording is unclear. Better "including at least one lowercase character, ..."

Reply:
Ok, i was confused by "password using ...", maybe better "... password choosing characters from at least all alphanumerics, including upper- and lowercase". Not sure though